eclair-core/src/main/scala/fr/acinq/eclair/wire/TlvTypes.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair.wire
+
+import scodec.bits.ByteVector
+
+/**
+  * Created by t-bast on 20/06/2019.
+  */
+
+// @formatter:off
+trait Tlv
```

Comment:
I see you didn't make this a `sealed trait` on purpose, that makes sense.

Reply:
Yes, it is currently necessary to define test tlv namespaces elsewhere...
Once we have enough real tlv types defined maybe we can switch the tests to use those real tlvs and remove the test ones and make this a sealed trait.